data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -114,22 +114,20 @@ then
 		--network-config-file=/assets/manifests/cluster-network-02-config.yml
 
 	cp etcd-bootstrap/manifests/* manifests/
-	cp etcd-bootstrap/bootstrap-manifests/etcd-member-pod.yaml /etc/kubernetes/manifests/
-
-	mkdir --parents /etc/kubernetes/static-pod-resources/etcd-member
-	cp tls/etcd-ca-bundle.crt /etc/kubernetes/static-pod-resources/etcd-member/ca.crt
-	# Temporarily ensure compatibility with both the new single cert secret and
-	# legacy multiple secrets. Support for copying the legacy secrets can be
-	# removed once https://github.com/openshift/cluster-etcd-operator/pull/544
-	# merges.
-	if [ -d "etcd-bootstrap/bootstrap-manifests/secrets/etcd-all-certs" ]
+
+	# If the etc-kubernetes path is present, all files and their paths are intended to be
+	# recursively copied to /etc/kubernetes.  This conditional ensures temporary backwards
+	# compatibility for the operator as it transitions to generating the tls assets it
+	# requires and owning where its rendered bootstrap assets should be copied to.
+	if [ -d "etcd-bootstrap/etc-kubernetes/" ]
 	then
-		# Copy the contents of the single aggregated cert secret if it is present.
-		cp --recursive etcd-bootstrap/bootstrap-manifests/secrets/etcd-all-certs /etc/kubernetes/static-pod-resources/etcd-member
+		cp --recursive etcd-bootstrap/etc-kubernetes/* /etc/kubernetes/
+		cp etcd-bootstrap/tls/* tls/
 	else
-		# Copy the contents of the legacy secrets
-		cp --recursive etcd-bootstrap/bootstrap-manifests/secrets/etcd-all-serving /etc/kubernetes/static-pod-resources/etcd-member
-		cp --recursive etcd-bootstrap/bootstrap-manifests/secrets/etcd-all-peer /etc/kubernetes/static-pod-resources/etcd-member
+		cp etcd-bootstrap/bootstrap-manifests/etcd-member-pod.yaml /etc/kubernetes/manifests/
+		mkdir --parents /etc/kubernetes/static-pod-resources/etcd-member
```

Comment:
should we take this time to align on the naming of static pod and friends? I know it's an extra step but I imagine we still have another installer change for 4.8

s/etcd-member/etcd-pod?
```
# TODO remove reference to etcd-member in 4.8
mkdir --parents /etc/kubernetes/static-pod-resources/etcd-pod
cp etcd-bootstrap/bootstrap-manifests/*-pod.yaml /etc/kubernetes/manifests/
ln -s /etc/kubernetes/static-pod-resources/etcd-pod /etc/kubernetes/static-pod-resources/etcd-member
```
We will need to work with SNO but then our render resources are more aligned with static-pod installer.

Reply:
You're commenting on the legacy codepath. The codepath that replaces it (L124-L125) ensures that the CEO has complete control over the paths it renders to in /etc/kubernetes. The following sequence of PRs are required:

- this PR (https://github.com/openshift/installer/pull/4686)enables compatibility with the new ceo rendering scheme
- https://github.com/openshift/cluster-etcd-operator/pull/546 implements the new rendering scheme
- https://github.com/openshift/installer/pull/4691 removes the compatibility path

Reply:
aha makes sense, thanks